packages/react-server/core/components/TheFold.js

```diff
@@ -0,0 +1,19 @@
+import {Component} from "react";
+
+export default class TheFold extends Component {
+	render() {
+		throw new Error("Something went wrong.  Trying to render the fold...");
+	}
+}
+
+TheFold.defaultProps = {
```

Comment:
Yeah, I actually thought about that.  AFAIK the reason `React` does that is to avoid injection attacks via JSON.  I don't see that as a concern here, so I opted to just keep it simple.